client/src/pages/SymptomsCheck.js

```diff
@@ -200,10 +203,21 @@ const AnswerButton = ({ children, onSelect }) => {
 const AnswerCheckbox = ({ text, content, checked, onSelect }) => {
   return (
     <AnswerStyles onClick={onSelect} className={checked && "selected"}>
-      <span>
-        {text}
-        {!!content && <h6>{content}</h6>}
-      </span>
+      <div>
+        <TextLabel size={theme.typography.size.mediun} block="true">
+          {text}
+        </TextLabel>
+        {!!content && (
```

Comment:
don't think you need `!!` here? 
I guess if you just omit it, it will also turn  `expression && ...` into a Boolean (falsy / truthy)

Reply:
Good catch! This was already there before and I overlooked it. Removed falsy now.